app/views/schools/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="govuk-heading-xl">New school</h1>
+
+<%= form_with model: @school ||= School.new, url: schools_path, local: true do |form| %>
```

Comment:
We will likely get rid of `local: true` once we cut out turbolinks and rails-ujs.